### PR TITLE
Changelog reminder

### PR DESCRIPTION
**Pull request by spydon**
# Description

Just adding an entry in the PR reminder list about the changelog file.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

If something is unclear, please submit the PR anyways and ask about what you thought was unclear.

- [x] This branch is based on `develop`
- [x] This PR is targeted to merge into `develop` (not `master`)
- [x] I have formatted my code with `flutter format`
- [x] I have made corresponding changes to the documentation
- [x] I have added examples for new features in `doc/examples`
- [x] The continuous integration (CI) is passing

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ If something is unclear, please submit the PR anyways and ask about what you tho
 
 - [ ] This branch is based on `develop`
 - [ ] This PR is targeted to merge into `develop` (not `master`)
+- [ ] I have added an entry on `[next]` in `CHANGELOG.md`
 - [ ] I have formatted my code with `flutter format`
 - [ ] I have made corresponding changes to the documentation
 - [ ] I have added examples for new features in `doc/examples`
```
Comment (erickzanardo) [inline .github/pull_request_template.md:25]: We could change to docs here, as the folder was renamed

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ If something is unclear, please submit the PR anyways and ask about what you tho
 
 - [ ] This branch is based on `develop`
 - [ ] This PR is targeted to merge into `develop` (not `master`)
+- [ ] I have added an entry on `[next]` in `CHANGELOG.md`
```
Comment (erickzanardo) [inline .github/pull_request_template.md:22]: Suggestion “under” instead of “on”

File: .github/pull_request_template.md (hunk 1 above)
Comment (spydon) [reply .github/pull_request_template.md]: Done!

File: .github/pull_request_template.md (hunk 2 above)
Comment (spydon) [reply .github/pull_request_template.md]: Done!

**Pull request merged**